spacepy/pybats/__init__.py

```diff
@@ -624,10 +624,10 @@ def _scan_bin_header(f, endchar, inttype, floattype):
     # Stash relevant values into the "info" dict:
     vals = readarray(f, dtype=header_fields_dtype, inttype=inttype)[0]
     for v, x in zip(['iter', 'runtime', 'ndim', 'nparam', 'nvar'], vals):
-        info[v] = x
+        info[v] = x.item()
 
     # Dimensionality may be negative to indicate non-uniform grid:
-    info['ndim'] = abs(info['ndim'])
+    info['ndim'] = np.abs(info['ndim'])
```

Comment:
If we're making everything Python integers, this should just stay abs, not np.abs. Sorry :(